apps/src/templates/projects/FeaturedProjectsTable.jsx

```diff
@@ -78,23 +78,53 @@ const nameFormatter = (projectName, {rowData}) => {
   return <a style={tableLayoutStyles.link} href={url} target="_blank">{projectName}</a>;
 };
 
+const unfeature = (channel) => {
+  var url = `/featured_projects/${channel}/unfeature`;
+  $.ajax({
+    url: url,
+    type:'PUT',
+    dataType:'json',
+  }).done(handleSuccess).fail(handleUnfeatureFailure);
+};
+
+const handleSuccess = () => {
+  window.location.reload(true);
+};
```

Comment:
seems like a good short term solution, though this will likely exacerbate any performance problems with the initial load as the number of featured projects grows large.